### ISSUE TITLE
Make conference name, website and support e-mail available in localization files

### ISSUE THREAD

**Issue by Amselfass** [DeutscheModelUnitedNations/munify-delegator#94]
Working on the localization files for the chatbot, I noticed that it would be really helpful to have the following 'variables' available for use in the text:

- Conference name (e.g. MUN-SH 2025)
- Conference website (e.g. https://mun-sh.de)
- Support e-mail (e.g. teilnehmende@mun-sh.de)

These values keep coming up and the text would be smoother if we could just insert them instead of awkwardly writing "the conference website".

If there's already a way to do this, feel free to close this and tell me where I can RTFM. :-)

**Comment by m1212e** [comment]
We could solve this with parameters: https://inlang.com/m/gerre34r/library-inlang-paraglideJs/usage
We should define some keywords e.g. "conferenceName" which then could be escaped by square brackets {} to become a parameter which lets us dynamically insert strings into the localization string.

E.g. "Welcome to the conference" would become "Welcome to {conferenceName}"

**Comment by TadeSF** [comment]
Just to make sure there is no missunderstanding of the feature: There is no global way to do this, since we have multiple conferences, etc. Every time you would want to use one of these, we would need to pass the value as a parameter into the language function.

I would suggest you create a PR that uses these parameters wherever and we fill them with the needed data afterwards :)

The Support Mail is not yet a field in the database, but I guess its a good Idea to add it @m1212e?

**Comment by m1212e** [comment]
Yes, we can add that, good idea! I think thats a good approach. I'll create a branch from this issue to work on that

**Comment by Amselfass** [comment]
So ... just to make sure I have expressed myself clearly:

I have already noticed that there is a way to import the text using
`m.ExampleMessage(animal: "bear")`
and then enter the text in the localization tool as
`"ExampleMessage": "The {animal} is my favourite animal."`

But that's a bit bothersome, because every time I want to use a parameter like this, I have to make a PR to change the code where the text is used.

I was wondering if there is a way to have something like "global variables" for the localization, so something like `{conferenceName}` could be freely used in the localization without having to change the code everywhere it's used.

**Comment by m1212e** [comment]
I'm afraid this is not possible since the localizations work on a global, app wide scale. The app allows for multiple conferences to exist in parallel so in these scenarios it might be tricky to find the correct conference name. I'm afraid the params are the only actual solution to this. If this bothers you, feel free to ask us about necessary code changes and go ahead when using non-existing parameters! Let us know if we can help you out!

**Comment by Amselfass** [comment]
Yeah, I already thought as much when I spelled out my request in detail. I will come back to your offer when necessary. :-)
